Packs/SpyCloudEnterpriseProtection/ReleaseNotes/1_0_5.md

```diff
@@ -3,9 +3,12 @@
 
 ##### SpyCloud Enterprise Protection Feed
 
+- Added support for **Fetch Limit** parameter that allows the number of incidents captured at a single time. by default this is set to 200.
+- Added support for **Domain Search** parameter that allows to filter watchlist on domains basis, if left empty, your full watchlist will be pulled.
```

Comment:
you can't update an already existing RN, please adda new Rn file and update the metadata file, you can do it by running `demisto-sdk update-release-notes -g` then when the RN file is generated, edit it with these 2 lines.

Reply:
Hi @merit-maita, We have made the changes as suggested by you as well as content-bot. Please review and provide your feedback